tests/process.rs

```diff
@@ -104,3 +104,22 @@ fn test_process_disk_usage() {
         p.disk_usage().written_bytes
     );
 }
+
+#[test]
+fn cpu_usage_is_not_nan() {
+    let mut system = sysinfo::System::new();
+    system.refresh_processes();
+
+    let first_5_pids = system.get_processes()
+        .iter()
+        .take(10)
+        .map(|(&pid, _)| pid)
+        .collect::<Vec<_>>();
+
+    first_5_pids.into_iter().for_each(|pid| {
+        system.refresh_process(pid);
+        if let Some(p) = system.get_process(pid) {
```

Comment:
I would suggest to have an `else` statement to that `if let`:
```
else {
    panic!("could not find pid")
}
```
Just to make sure that the only way for the test to pass is that the assertion succeeded

Reply:
I had the issue on windows that some process might be missing (dark magic). I can add a check that at least one process was checked though, good idea!